### PR TITLE
#165273586 Time to read an article

### PR DESCRIPTION
**Pull request by bisonlou**
#### What does this PR do?
- Adds estimated time taken to read an article.

#### Description of the task to be completed?
- Add an estimation of time taken to read an article in minutes.

#### How should this be manually tested?
- Clone the Github repository - `git clone https://github.com/andela/ah-backend-dojo.git`
- Navigate into the repository - `cd ah-backend-dojo`
- Create a virtual environment - `python3 -m venv [virtual_environment_name]`
- Activate the virtual environment - macOS: `source [virtual_environment_name]/bin/activate`
- Install the dependencies from the requirements file - `pip install -r requirements.txt`
-  Create a `.env` file, copy and paste in environment variables setup in the `.env_sample` file, in the blank quotes, add your PostgreSQL database credentials.
- Make the migrations - `python manage.py makemigrations`
- Migrate - `python manage.py migrate`
- Run the server - `python manage.py runserver`
- Run your postman application
- URL: http://127.0.0.1:8000/api/articles/ to view articles

#### Any background context you want to provide?
- Currently, users have no way of telling how long it takes to read an article

#### What are the relevant pivotal tracker stories
- [#165273586](https://www.pivotaltracker.com/story/show/165273586)

#### Screenshots
- Estimated time to read an article in minutes
image

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -30,14 +34,17 @@ psycopg2-binary==2.8.2
 py==1.8.0
 PyJWT==1.7.1
 pylint==2.3.1
+pyquery==1.4.0
```
Comment (nadralia) [inline requirements.txt:37]: @bisonlou  is this library necessary for now?.

Comment (bisonlou) [reply requirements.txt:37]: I have no Idea how they came about. I will investigate to see if the app can run without them.

**Review by nadralia — APPROVED** [review]

**Review by kalsmic — APPROVED** [review]
@bisonlou, kudos, time to read an article works like magic. It's amazing how did not reinvent the wheel.

**Review by JamesMudidi — APPROVED** [review]
@bisonlou, Thumbs  up on your work

**Review by ZackAtama — APPROVED** [review]
Great work Lou

**Pull request merged**